commands/@.js

```diff
@@ -17,6 +17,28 @@ function isAdmin(){
   return user.id == admin_id
 }
 
+function canRemoveByAngryPoints(value) {
+  if (value > 0) {
+    return false
+  }
+  var pvalue = -value // convert amount to positive
+  var AngryPointsLimit = Libs.ResourcesLib.userRes("AngryPointsLimit")
+  if (pvalue > AngryPointsLimit.value()) {
+    return false
+  }
+
+  var AngryPoints = Libs.ResourcesLib.userRes("AngryPoints")
```

Comment:
you have bad name for AngryPoints here. 
It is maxAngryPointsLimit

var maxAngryPointsLimit = Libs.ResourcesLib.userRes("maxAngryPointsLimit")